txlib/api/translations.py

```diff
@@ -22,12 +22,21 @@ def _create(self, **kwargs):
         object's case, it needs to be `PUT`, while in the BaseModel is `POST`
         """
         path = self._construct_path_to_collection()
+        content = kwargs['content']
+        is_binary = Translation.is_binary(content)
```

Comment:
Based on my comment about the `is_binary()` method, I suggest this gets changed to:
```
is_binary = not isinstance(content, str)
```